src/core/Akka/Util/MatchHandler/MatchBuilder.cs

```diff
@@ -171,21 +171,6 @@ public PartialAction<TItem> Build()
             return partialAction;
         }
 
-#if !CORECLR
-        /// <summary>
-        /// TBD
-        /// </summary>
-        /// <param name="typeBuilder">TBD</param>
-        /// <param name="methodName">TBD</param>
-        /// <param name="attributes">TBD</param>
-        /// <returns>TBD</returns>
-        public void BuildToMethod(TypeBuilder typeBuilder, string methodName, MethodAttributes attributes = MethodAttributes.Public | MethodAttributes.Static)
```

Comment:
Byterot, these codes were never compiled before the cleanup (since we moved to single `netstandard2.0` platform target)